### DISCUSSION TITLE
No memory or reach tenant memory limit

### DISCUSSION THREAD

**Discussion by iflmyz** [oceanbase/oceanbase#95] [Q&A] [answered]
环境：采用docker安装，给的内存8G

操作：将sql(100M)中的文件导入oceanbase中

问题：ERROR 4013 (HY001) at line 920: No memory or reach tenant memory limit

**Comment by simonjoylet** [comment]
may be the speed of insert is larger than the speed of dump.
try parameters like this:
alter system set _enable_parallel_minor_merge = true  tenant your_tenant_name;
alter system set _mini_merge_concurrency = 10;
alter system set freeze_trigger_percentage = 30;

**Comment by nroskill** [comment]
alter system set writing_throttling_trigger_percentage=80；
try this

**Comment by amber-moe** [comment, marked as answer]
![image](https://user-images.githubusercontent.com/42762957/120765528-e3343280-c54b-11eb-961a-d90120c8b360.png)
